<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAafMainDialog</class>
 <widget class="QDialog" name="QtAafMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AminoAcidFighter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GameAminoAcidFighter.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="interactive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_left">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Rotate &amp;left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_auto_rotate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Auto rotate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_right">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Rotate &amp;right</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GameAminoAcidFighter.qrc"/>
 </resources>
 <connections/>
</ui>
